arm-notificationhubs/2016-03-01/swagger/notificationhubs.json

```diff
@@ -1757,7 +1818,6 @@
           "description": "Resource Id"
         },
         "name": {
-          "readOnly": true,
```

Comment:
Not sure why would you remove it, name from the body does not suppose to be used by RP, so this suppose to be read only.